web/styles/left_sidebar.css

```diff
@@ -1361,7 +1361,13 @@ li.topic-list-item {
     overflow: hidden;
 }
 
-.topic-list-filter {
+.pm-search-section {
+    margin-left: 10px;
+    padding-bottom: 5px;
+}
+
+.topic-list-filter,
+.pm-search-section {
```

Comment:
I used `.pm-search-section` of the parent div of the input element instead of `.pm-list-filter` class of the input element because the parent of the input element is a flexbox and the width property won't make a difference visually. I could have used max-width to achieve the same but then sharing the property would have not been possible.